### PR TITLE
Updated Encoding parameter and YAML for Select-String

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Related to Issue #2000
Updated Encoding parameter and YAML for Select-String
Style updates
Paragraph reflows
Fixed links

Acrolinx scores
Original version: 86
Updated version: 94

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[faba0bf](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/faba0bfa10ce7c46795c2a77c272a4ebc23d9eea)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[faba0bf](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/faba0bfa10ce7c46795c2a77c272a4ebc23d9eea)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[faba0bf](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/faba0bfa10ce7c46795c2a77c272a4ebc23d9eea)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;3.0/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :52:(138) &#39;This command performs a case-sensitive match of th...&#39;
 Expect Heading

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;4.0/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :52:(138) &#39;This command performs a case-sensitive match of th...&#39;
 Expect Heading

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;5.0/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :52:(138) &#39;This command performs a case-sensitive match of th...&#39;
 Expect Heading

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;5.1/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :52:(138) &#39;This command performs a case-sensitive match of th...&#39;
 Expect Heading

<a name="reference%2F6%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;6/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :52:(138) &#39;This command performs a case-sensitive match of th...&#39;
 Expect Heading

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C8%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902082344126197-3679%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JUefRw4OLdLM9sKyjGGgZ%2B1e3oFs5KDW2FGusB6fKfM%3D&st=2019-02-08T23%3A52%3A09Z&se=2019-03-11T23%3A57%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[46cd610](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/46cd61040aa1c5b4bdfce999ed347c78cb2ef818)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[46cd610](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/46cd61040aa1c5b4bdfce999ed347c78cb2ef818)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[46cd610](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/46cd61040aa1c5b4bdfce999ed347c78cb2ef818)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)
[reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md) | :x:Error | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Utility%2FSelect-String.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;3.0/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :646:(6801) &#39;.

&#39;
 Expect hyperlink, but got .

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;4.0/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :646:(6801) &#39;.

&#39;
 Expect hyperlink, but got .

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;5.0/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :646:(6801) &#39;.

&#39;
 Expect hyperlink, but got .

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;5.1/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :646:(6801) &#39;.

&#39;
 Expect hyperlink, but got .

<a name="reference%2F6%2FMicrosoft.PowerShell.Utility%2FSelect-String.md"></a>
### [reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md)
 - **[Error]** Unable to load file &#39;6/Microsoft.PowerShell.Utility/Select-String.md&#39; via processor &#39;AzureCliDocumentProcessor&#39;: :648:(6772) &#39;.

&#39;
 Expect hyperlink, but got .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C9%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902090018408747-3679%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FycH0KLBRu4XahMPy62K9oySWhfj43mbzhrxOy%2FsVtQ%3D&st=2019-02-09T00%3A26%3A11Z&se=2019-03-12T00%3A31%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b12ae4a](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/b12ae4ae5201bbc1c250ab9bcc704b03513fcdcc)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b12ae4a](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/b12ae4ae5201bbc1c250ab9bcc704b03513fcdcc)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b12ae4a](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/b12ae4ae5201bbc1c250ab9bcc704b03513fcdcc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-3.0&branch=pr-en-us-3679) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-4.0&branch=pr-en-us-3679) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.0&branch=pr-en-us-3679) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.1&branch=pr-en-us-3679) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-6&branch=pr-en-us-3679) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C9%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902090046227281-3679%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=uu73FJwSKbpgN0PRCpAbdtXbWGKAdHOXlnot9giA51A%3D&st=2019-02-09T00%3A54%3A21Z&se=2019-03-12T00%3A59%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b95614b](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/b95614bc7020a044ee6cd4a4908b0cf3f463ed60)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b95614b](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/b95614bc7020a044ee6cd4a4908b0cf3f463ed60)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[b95614b](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/b95614bc7020a044ee6cd4a4908b0cf3f463ed60)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-3.0&branch=pr-en-us-3679) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-4.0&branch=pr-en-us-3679) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.0&branch=pr-en-us-3679) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.1&branch=pr-en-us-3679) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-6&branch=pr-en-us-3679) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902111541080974-3679%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bz9yYXysYHVffBevGOVED9OKbC6IxS4ye7Ha3qxXhHs%3D&st=2019-02-11T15%3A50%3A56Z&se=2019-03-14T15%3A55%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Utility/Select-String.md [hunk 1]
```diff
@@ -399,12 +501,11 @@ Accept wildcard characters: False
...
-Specifies the path to the files to be searched.
-Wildcards are permitted.
-The default location is the local directory.
+Specifies the path to the files to search. Wildcards are permitted. The default location is the
+local directory.
 
-Specify files in the directory, such as "log1.txt", "*.doc", or "*.*".
-If you specify only a directory, the command fails.
+Specify files in the directory, such as `log1.txt`, `*.doc`, or `*.*`. If you specify only a
+directory, the command will fail.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Select-String.md]: Strike *will*. Passive voice.

If you specify only a directory, the command fails.

Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Utility/Select-String.md]: Thanks. I made the changes and updated the PR.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[1875fb3](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/1875fb39ce2c9512b6191a76e8e5e4e86be98a5c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[1875fb3](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/1875fb39ce2c9512b6191a76e8e5e4e86be98a5c)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[1875fb3](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/1875fb39ce2c9512b6191a76e8e5e4e86be98a5c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-3.0&branch=pr-en-us-3679) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-4.0&branch=pr-en-us-3679) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.0&branch=pr-en-us-3679) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.1&branch=pr-en-us-3679) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-6&branch=pr-en-us-3679) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902111954187133-3679%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=22Wx%2FRbgtNdC5LJu7yD1npjNXD65Ly%2B4rIixtWpIlVE%3D&st=2019-02-11T20%3A02%3A42Z&se=2019-03-14T20%3A07%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[89442e0](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/89442e0f35c066d44a336a7e01dda823c3a22178)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[89442e0](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/89442e0f35c066d44a336a7e01dda823c3a22178)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 51 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[89442e0](https://github.com/davidsmatlak/PowerShell-Docs/pull/3679/commits/89442e0f35c066d44a336a7e01dda823c3a22178)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/3.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-3.0&branch=pr-en-us-3679) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/4.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-4.0&branch=pr-en-us-3679) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.0/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.0&branch=pr-en-us-3679) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/5.1/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-5.1&branch=pr-en-us-3679) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Select-String.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-selectstring/reference/6/Microsoft.PowerShell.Utility/Select-String.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/select-string?view=powershell-6&branch=pr-en-us-3679) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902112056459812-3679%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7KdMx8CVHUlkteHji70hbLSQA%2Ft8QYnhiyT7FeFir4I%3D&st=2019-02-11T21%3A06%3A36Z&se=2019-03-14T21%3A11%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**